examples/pytorch/geniepath/ppi.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+
+import numpy as np
+import torch as th
+import torch.optim as optim
+from dgl.data import PPIDataset
+from dgl.dataloading import GraphDataLoader
+from sklearn.metrics import f1_score
+
+from model import GeniePath, GeniePathLazy
+
+
+def evaluate(model, loss_fn, dataloader, device='cpu'):
+    loss = 0
+    f1 = 0
+    num_blocks = 0
+    for subgraph in dataloader:
+        subgraph = subgraph.to(device)
+        label = subgraph.ndata['label'].to(device)
+        feat = subgraph.ndata['feat']
+        logits = model(subgraph, feat)
+
+        # compute loss
+        loss += loss_fn(logits, label).item()
+        predict = np.where(logits.data.cpu().numpy() >= 0., 1, 0)
```

Comment:
Normally for binary classification, we use torch.argmax to get the predict 0 or 1. Any specific reasons here are you using np.where?

Reply:
PPI dataset has more than two classes, so I think np.where is necessary.

Reply:
ok